templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/list_tags.html.twig

```diff
@@ -364,11 +364,11 @@
             </div>
 
             {# import #}
-            <dp-tooltip-icon
+            <dp-contextual-help
                 class="float-right"
-                icon="fa-question-circle"
+                icon="question"
```

Comment:
The `icon` prop of DpContextualHelp defaults to `question`, so this can be removed here.

Reply:
Done here [be33bc2](https://github.com/demos-europe/demosplan-core/pull/2376/commits/be33bc2d4b3ad5fce405dfbb868049c07e0673ee)